src/05-glue/04-optparse.md

```diff
@@ -254,7 +254,7 @@ satisfy the first law. `fmap id = id` means that
 clearly see that `mapMaybe id (Just x) == Nothing`.
 
 This is a good example of how Haskell doesn't help us make sure the laws
-are satisfied, and why they are important. Unlawful `Functor` instances
+are satisfied, and why they are important? Unlawful `Functor` instances
```

Comment:
```suggestion
are satisfied, and why they are important. Unlawful `Functor` instances
```
A period is more appropriate here because we are not actually asking a question.